### PR TITLE
Tm 43 submit application

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/IOpportunitiesService.cs [hunk 1]
```diff
@@ -10,8 +10,10 @@ public interface IOpportunitiesService
         Task<List<OpportunityDto>> GetAllOpportunities();
         Task<OpportunityDto> GetOpportunity(int id);
         Task<GetContactDetailsResponse> GetContactDetails(long accountId, int pledgeId);
-        Task<ApplicationDetailsDto> GetApplicationDetails(int id);
+		Task<ApplicationDetailsDto> GetApplicationDetails(long accountId, int id);
```
Comment (ConorTill) [inline src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/IOpportunitiesService.cs:13]: Weird tab lengths present in a couple places, doesn't show up in visual studio so maybe it's fine

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/OpportunitiesService.cs [hunk 2]
```diff
@@ -91,6 +92,27 @@ public async Task<GetContactDetailsResponse> GetContactDetails(long accountId, i
             }
 
             return getContactDetailsResult;
+        }
+		
+		public async Task<GetConfirmationResponse> GetConfirmation(long accountId, int opportunityId)
```
Comment (ConorTill) [inline src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/OpportunitiesService.cs:97]: More spacing weirdness

File: src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Orchestrators/OpportunitiesOrchestratorTests.cs [hunk 3]
```diff
@@ -470,6 +473,82 @@ public async Task GetApplyViewModel_Returns_Empty_Value_For_False_HasTrainingPro
 
             Assert.AreEqual("No", result.HaveTrainingProvider);
         }
+		
+		[Test]
```
Comment (ConorTill) [inline src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Orchestrators/OpportunitiesOrchestratorTests.cs:477]: More spacing weirdness

**Review by ConorTill — APPROVED** [review]

File: src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/IOpportunitiesService.cs (hunk 1 above)
Comment (chrisfoster76) [reply src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/IOpportunitiesService.cs:13]: Odd, I'll try to reformat and see if it goes away

**Review by DannyGB — APPROVED** [review]

**Pull request merged**